src/kite/messagescrubber.js

```diff
@@ -15,7 +15,7 @@ export default class MessageScrubber {
       authentication: this.kite.options.auth,
       withArgs: params,
       responseCallback(response) {
-        const { error: rawErr, result } = response
+        const { error: rawErr, result } = response || {}
```

Comment:
using default arg here `responseCallback(response = {})` would be more idiomatic IMO.